src/lib/gssapi/mechglue/g_set_context_flags.c

```diff
@@ -0,0 +1,53 @@
+/* #pragma ident	                                                    */
+
+/*
+ *  glue routine for gss_set_context_flags
+ */
+
+#include "mglueP.h"
+#ifdef HAVE_STDLIB_H
+#include <stdlib.h>
+#endif
+#include <string.h>
+#include <errno.h>
+#include <time.h>
+
+#ifndef LEAN_CLIENT
+
+
+/*
+ * Defined:
+ * https://tools.ietf.org/html/draft-ietf-kitten-channel-bound-flag-01
+ *
+ * Section 2.2
+ *
+ * See src/lib/gssapi/generic/gssapi_ext.h for type definitions.
+ */
+OM_uint32 KRB5_CALLCONV
+gss_set_context_flags(OM_uint32 *minor_status, gss_ctx_id_t context,
+                      uint64_t req_flags, uint64_t ret_flags)
```

Comment:
The I-D has only three arguments for this function.  There should be no way to _set_ `ret_flags` directly.